pimpmycause/pimpmycause/templates/profiles/profile.html

```diff
@@ -6,71 +6,93 @@
+                                        {{ profile_image_upload_form.media }}
+                                        {{ profile_image_upload_form.as_p }} -->
                                     </div>
-                                    <div class="small-12 columns">
+                                    <div class="small-6 large-6 medium-6 columns">
                                         {{ profile_update_form.name.errors }}
                                         <label for="name"> Name </label>
                                         <input type="text" name="name" placeholder="Name" value='{{ profile_update_form.name.value|default_if_none:"" }}' required>
-                                    </div>
-                                    <div class="small-12 columns">
+
                                         {{ profile_update_form.surname.errors }}
                                         <label for="surname"> Surname </label>
                                         <input type="text" name="surname" placeholder="Surname" value='{{ profile_update_form.surname.value|default_if_none:"" }}' required>
                                     </div>
-                                    <div class="small-12 columns">
-                                        {{ profile_update_form.phone.errors }}
-                                        <label for="phone"> Phone </label>
-                                        <input type="text" name="phone" placeholder="Phone" value='{{ profile_update_form.phone.value|default_if_none:"" }}' required>
-                                    </div>
-                                    <div class="small-12 columns">
+                                </div>
+                            </div>
+                            <div class="row">
+                                <h2> Social</h2>
+                            </div>
+                            <div class="row">
+                                <div class="large-6 medium-6 small-12 columns">
+                                    {{ profile_update_form.linkedin.errors }}
+                                    <label for="linkedin"> LinkedIn </label>
+                                    <input type="url" name="linkedin" placeholder="LinkedIn" value='{{ profile_update_form.linkedin.value|default_if_none:"" }}' required>
+
+                                    {{ profile_update_form.twitter.errors }}
+                                    <label for="twitter"> Twitter </label>
+                                    <input type="url" name="twitter" placeholder="Twitter" value='{{ profile_update_form.twitter.value|default_if_none:"" }}' required>
+
+                                    {{ profile_update_form.website.errors }}
+                                    <label for="website"> Website </label>
+                                    <input type="url" name="website" placeholder="www.example.com" value='{{profile_update_form.website.value|default_if_none:""}}' required>
```

Comment:
could we change the placeholder to say something like http://www.example.com ? or no need ?